staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -245,6 +247,7 @@ func (le *LeaderElector) renew(ctx context.Context) {
 			klog.V(5).Infof("successfully renewed lease %v", desc)
 			return
 		}
+		leaderGauge.WithLabelValues(le.config.Name).Set(0.0)
```

Comment:
Out of curiosity, what is the value of `le.config.Name`?

Reply:
It depends on the lease in question. So for the KCM its "kube-controller-manager" and for the scheduler its "kube-scheduler".